CHANGELOG.md

```diff
@@ -11,6 +11,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### New features
 
+* `jj log --summary --patch` now shows both summary and diff outputs.
```

Comment:
IIUC, we also allow e.g. `jj diff -s --git` now. I'm not sure that's useful enough to be worth mentioning here. But maybe we should have a test for one of those cases?

Reply:
Good point. Added test for that.

Reply:
Looks like you added one for `log`. Did you mean for that to be for `diff`? I don't mind having one for `log`, but we also support `jj diff -s --git`, right?

Reply:
Oops, fixed.

Reply:
Looks good now. Thanks for adding both tests :)